lib/logstash/filters/translate.rb

```diff
@@ -69,11 +70,11 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   # (in seconds) logstash will check the YAML file for updates.
   config :refresh_interval, :validate => :number, :default => 300
   
-  # The destination field you wish to populate with the translated code. The default
-  # is a field named `translation`. Set this to the same value as source if you want
-  # to do a substitution, in this case filter will allways succeed. This will clobber
-  # the old value of the source field! 
-  config :destination, :validate => :string, :default => "translation"
+  # The destination field you wish to populate with the translated code. If not set
+  # [source field]_translation will the destination field. Set this to the same value
+  # as source and set override to true if you want to do a substitution, in this case
+  # filter will allways succeed. This will clobber the old value of the source field!
+  config :destination, :validate => :string
```

Comment:
I understand this changes are having an impact on the way people is using this plugin, basically changing the public interface. This will require us to raise the major version of this plugin, can you do so inside the gemspec?